### ISSUE TITLE
getting panic error on deploy command

### ISSUE THREAD

**Issue by Shareed2k** [loft-sh/devspace#1303]
**What happened?**  
tried to run `devspace deploy` and getting this panic

```sh
devspace deploy
panic: reflect.Set: value of type map[interface {}]interface {} is not assignable to type runtime.Object [recovered]
	panic: reflect.Set: value of type map[interface {}]interface {} is not assignable to type runtime.Object

goroutine 1 [running]:
gopkg.in/yaml%2ev2.handleErr(0xc000fcd7c0)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/yaml.go:249 +0x85
panic(0x56da760, 0xc00143e260)
	/Users/runner/hostedtoolcache/go/1.15.7/x64/src/runtime/panic.go:969 +0x1b9
reflect.Value.assignTo(0x5755fa0, 0xc0013fa1e0, 0x15, 0x5a78b89, 0xb, 0x57abc80, 0xc00143e250, 0x5755fa0, 0x0, 0xc0013fa1e0)
	/Users/runner/hostedtoolcache/go/1.15.7/x64/src/reflect/value.go:2435 +0x405
reflect.Value.Set(0x57abc80, 0xc00143e250, 0x194, 0x5755fa0, 0xc0013fa1e0, 0x15)
	/Users/runner/hostedtoolcache/go/1.15.7/x64/src/reflect/value.go:1564 +0xbd
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc0007967e0, 0xc0001a16c0, 0x57abc80, 0xc00143e250, 0x194, 0x57abc80)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:635 +0x8df
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0007967e0, 0xc0001a16c0, 0x57abc80, 0xc00143e250, 0x194, 0x194)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:372 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc0007967e0, 0xc0001a1500, 0x575b640, 0xc0013ab070, 0x195, 0x575b640)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:676 +0x479
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0007967e0, 0xc0001a1500, 0x575b640, 0xc0013ab070, 0x195, 0xc0013ab070)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:372 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).mappingStruct(0xc0007967e0, 0xc0001a09a0, 0x591e120, 0xc0013ab000, 0x199, 0x5798860)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:767 +0xa33
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc0007967e0, 0xc0001a09a0, 0x591e120, 0xc0013ab000, 0x199, 0x591e120)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:626 +0xa67
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0007967e0, 0xc0001a09a0, 0x5788220, 0xc0004c2048, 0x196, 0x196)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:372 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc0007967e0, 0xc0001a08c0, 0x5759360, 0xc00051b6b0, 0x195, 0x5759360)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:676 +0x479
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0007967e0, 0xc0001a08c0, 0x5759360, 0xc00051b6b0, 0x195, 0xc00051b6b0)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:372 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).mappingStruct(0xc0007967e0, 0xc0001a0150, 0x591e240, 0xc00051b680, 0x199, 0x5798860)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:767 +0xa33
gopkg.in/yaml%2ev2.(*decoder).mapping(0xc0007967e0, 0xc0001a0150, 0x591e240, 0xc00051b680, 0x199, 0x591e240)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:626 +0xa67
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0007967e0, 0xc0001a0150, 0x591e240, 0xc00051b680, 0x199, 0x0)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:372 +0x1b6
gopkg.in/yaml%2ev2.(*decoder).document(0xc0007967e0, 0xc0001a00e0, 0x591e240, 0xc00051b680, 0x199, 0x44d23a7)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:384 +0x7c
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0007967e0, 0xc0001a00e0, 0x591e240, 0xc00051b680, 0x199, 0x199)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/decode.go:360 +0x250
gopkg.in/yaml%2ev2.unmarshal(0xc0010c4000, 0x3f5b0, 0x4c000, 0x5849ca0, 0xc00051b680, 0x0, 0x0, 0x0)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/yaml.go:148 +0x33d
gopkg.in/yaml%2ev2.Unmarshal(...)
	/Users/runner/work/devspace/devspace/vendor/gopkg.in/yaml.v2/yaml.go:81
github.com/devspace-cloud/devspace/pkg/devspace/config/versions/util.Convert(0x5651820, 0xc0004c21d0, 0x5849ca0, 0xc00051b680, 0x199, 0x589a001)
	/Users/runner/work/devspace/devspace/pkg/devspace/config/versions/util/convert.go:11 +0x85
github.com/devspace-cloud/devspace/pkg/devspace/kubectl/util.NewClientByContext(0x0, 0x0, 0x0, 0x0, 0xc001007900, 0x5e41de0, 0x71f8548, 0xc001363c50, 0xc001007a40, 0x469062b, ...)
	/Users/runner/work/devspace/devspace/pkg/devspace/kubectl/util/util.go:26 +0xe7
github.com/devspace-cloud/devspace/pkg/devspace/kubectl.NewClientFromContext(0x0, 0x0, 0x0, 0x0, 0x0, 0x5e41de0, 0x71f8548, 0xc00138e370, 0xc001363ad0, 0x0, ...)
	/Users/runner/work/devspace/devspace/pkg/devspace/kubectl/client.go:79 +0x8d
github.com/devspace-cloud/devspace/pkg/util/factory.(*DefaultFactoryImpl).NewKubeClientFromContext(0x71f8548, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xe0)
	/Users/runner/work/devspace/devspace/pkg/util/factory/factory.go:154 +0x78
github.com/devspace-cloud/devspace/cmd.(*DeployCmd).Run(0xc0005ae3c0, 0x5e4e9e0, 0x71f8548, 0x71f8548, 0x0, 0x0, 0xc000ea3b80, 0x71f8548, 0x0, 0x0, ...)
	/Users/runner/work/devspace/devspace/cmd/deploy.go:135 +0x50c
github.com/devspace-cloud/devspace/cmd.NewDeployCmd.func1(0xc000ea3b80, 0x71f8548, 0x0, 0x0, 0x0, 0x0)
	/Users/runner/work/devspace/devspace/cmd/deploy.go:74 +0xca
github.com/spf13/cobra.(*Command).execute(0xc000ea3b80, 0x71f8548, 0x0, 0x0, 0xc000ea3b80, 0x71f8548)
	/Users/runner/work/devspace/devspace/vendor/github.com/spf13/cobra/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000f6580, 0x71f8548, 0xc0000f6580, 0x0)
	/Users/runner/work/devspace/devspace/vendor/github.com/spf13/cobra/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/runner/work/devspace/devspace/vendor/github.com/spf13/cobra/command.go:887
github.com/devspace-cloud/devspace/cmd.Execute()
	/Users/runner/work/devspace/devspace/cmd/root.go:99 +0x7d
main.main()
	/Users/runner/work/devspace/devspace/main.go:20 +0xf6
```

**What did you expect to happen instead?**  
Deploy the project 

**Local Environment:**  
- DevSpace Version: 5.7.3
- Operating System: mac
- Deployment method: helm

**Kubernetes Cluster:**  
- Kubernetes Version: v1.20.2 minikube

/kind bug

**Comment by FabianKramm** [comment]
@Shareed2k thanks for reporting this issue! Yes there seems to be an issue with minikube and kubernetes version 1.20.2, thanks for catching that! We'll fix this in the next version

**Comment by FabianKramm** [comment]
@Shareed2k should be fixed with v5.7.4
